### PR TITLE
vstart.sh: Remove duplicate global section

### PR DESCRIPTION
**Pull request by iliul**
remove duplicate global section，move `$extra_conf` to the first global section

### CODE REVIEW

**Comment by iliul** [comment]
@joscollin Please help take a look，Thanks

File: src/vstart.sh [hunk 1]
```diff
@@ -440,6 +440,7 @@ prepare_conf() {
         filestore fd cache size = 32
         run dir = $CEPH_OUT_DIR
         enable experimental unrecoverable data corrupting features = *
+$extra_conf
```
Comment (tchaikov) [inline src/vstart.sh]: you might want to make sure `$extra_conf` is able to override the settings in all sections. for example, the default settings of `[client]` will override that the `$extra_conf` with your change.

Comment (iliul) [reply src/vstart.sh]: @tchaikov do you expect me add a `$extra_conf` to `[client]` section? To make sure add extra config parameters to all sections.

**Comment by liewegas** [comment]
see de4474acbd7d3e2d0d4ec511680bfedb59b0a462 .. this was deliberately at the end so that it overrides the stuff that is already in there.  I think you're probably right that this particular line can move back, but please test and verify that overriding a config that is in [global] already with -o 'foo = bar' will work correctly.

**Comment by liewegas** [comment]
and yeah, let's add it to [client] too

**Comment by iliul** [comment]
@liewegas added `$extra_conf` to `[client]` section, and it works fine in my local envionment.

**Pull request merged**